src/Stratis.Bitcoin.IntegrationTests/BlockStore/ProofOfWorkSpendingSpecification_Steps.cs

```diff
@@ -0,0 +1,131 @@
+﻿using System;
+using System.Linq;
+using FluentAssertions;
+using NBitcoin;
+using Stratis.Bitcoin.Features.Consensus;
+using Stratis.Bitcoin.Features.Wallet;
+using Stratis.Bitcoin.IntegrationTests.EnvironmentMockUpHelpers;
+using Xunit;
+
+namespace Stratis.Bitcoin.IntegrationTests.BlockStore
+{
+    public partial class ProofOfWorkSpendingSpecification
```

Comment:
Hmm this test seems to be very general `ProofOfWorkSpending` you might find that other cases need to create more then one nodes.
If this is a very specific specification then my preference is to keep related code in the same file as the test (maybe an internal class?).

Reply:
The separate partial is literally just to make the file you read most of the time (The specification) as clean as it possibly can be - the counter argument being hiding the code away in the steps file - but on balance I like as many tricks as possible to keep the readable file readable. Happy to hear others opinions on this though.